2011/12/10 01:24:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\xampp\htdocs\YiiRoot\framework\db\CDbConnection.php:364
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\CDbConnection.php(313): CDbConnection->open()
#1 C:\xampp\htdocs\YiiRoot\framework\db\CDbConnection.php(291): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\YiiRoot\framework\base\CModule.php(372): CDbConnection->init()
#3 C:\xampp\htdocs\YiiRoot\framework\base\CModule.php(86): CModule->getComponent('db')
#4 C:\xampp\htdocs\YiiRoot\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\xampp\htdocs\YiiRoot\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\xampp\htdocs\YiiRoot\framework\gii\CCodeGenerator.php(149): CFormModel->__construct()
#7 C:\xampp\htdocs\YiiRoot\framework\gii\CCodeGenerator.php(61): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=gii/model
2011/12/10 01:24:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in C:\xampp\htdocs\YiiRoot\framework\db\CDbConnection.php:364
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\CDbConnection.php(313): CDbConnection->open()
#1 C:\xampp\htdocs\YiiRoot\framework\db\CDbConnection.php(291): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\YiiRoot\framework\base\CModule.php(372): CDbConnection->init()
#3 C:\xampp\htdocs\YiiRoot\framework\base\CModule.php(86): CModule->getComponent('db')
#4 C:\xampp\htdocs\YiiRoot\framework\gii\generators\model\ModelCode.php(54): CModule->__get('db')
#5 C:\xampp\htdocs\YiiRoot\framework\web\CFormModel.php(40): ModelCode->init()
#6 C:\xampp\htdocs\YiiRoot\framework\gii\CCodeGenerator.php(149): CFormModel->__construct()
#7 C:\xampp\htdocs\YiiRoot\framework\gii\CCodeGenerator.php(61): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(278): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('gii/model')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=gii/model
2011/12/10 01:35:01 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'globaldebtmgm.tbl_debtor_negotiator' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_debtor_negotiator`.
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/10 01:35:41 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'globaldebtmgm.tbl_progam_infor' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_progam_infor`.
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/10 01:35:53 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'globaldebtmgm.tbl_program_infor' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_program_infor`.
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/10 01:39:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/debtor".' in C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('gii/debtor')
#1 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/globaldebtmgm/index.php?r=gii/debtor
2011/12/11 21:21:23 [error] [exception.CException] exception 'CException' with message 'CActiveForm and its behaviors do not have a method or closure named "checkBoxEx".' in C:\xampp\htdocs\YiiRoot\framework\base\CComponent.php:269
Stack trace:
#0 C:\xampp\htdocs\globaldebtmgm\protected\views\user\_form.php(35): CComponent->__call('checkBoxEx', Array)
#1 C:\xampp\htdocs\globaldebtmgm\protected\views\user\_form.php(35): CActiveForm->checkBoxEx(Object(User), 'remember_me')
#2 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\globaldebtmgm\protected\views\user\create.php(15): CController->renderPartial('_form', Array)
#6 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#10 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(78): CController->render('create', Array)
#11 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#12 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#15 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#17 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#19 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#21 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#22 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#23 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#24 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 21:51:35 [warning] [application] Failed to set unsafe attribute "salt" of "User".
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (71)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 21:51:35 [warning] [application] Failed to set unsafe attribute "updated_at" of "User".
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (71)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 21:52:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "cuser".' in C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('cuser')
#1 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/globaldebtmgm/index.php?r=cuser
2011/12/11 22:01:42 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (72)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:01:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(72): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:01:59 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (72)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:01:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(72): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:02:06 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (72)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:02:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(72): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:02:39 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (72)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:02:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(72): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:02:42 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (72)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:02:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(72): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:07:39 [error] [php] Indirect modification of overloaded property User::$attributes has no effect (C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php:67)
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): UserController->runAction()
#1 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#2 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): UserController->filterAccessControl()
#4 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#5 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#6 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): UserController->run()
#8 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController()
#9 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#10 C:\xampp\htdocs\globaldebtmgm\index.php(13): CWebApplication->run()
REQUEST_URI=/globaldebtmgm/index.php?r=user/create
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (67)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:08:16 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`, `updated_at`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (70)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:08:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`, `updated_at`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(70): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:08:24 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`, `updated_at`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (70)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:08:24 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`, `updated_at`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(70): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:08:46 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (69)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:08:46 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(69): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:08:50 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (69)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:08:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(69): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:10:25 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (67)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:10:25 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `created_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(67): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:31:53 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `updated_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4).
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (72)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:31:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'admin' for key 'username_2'. The SQL statement executed was: INSERT INTO `tbl_user` (`remember_me`, `updated_at`, `username`, `password`, `email`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4)' in C:\xampp\htdocs\YiiRoot\framework\db\CDbCommand.php:336
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1017): CDbCommand->execute()
#1 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(72): CActiveRecord->save()
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:39:43 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'globaldebtmgm.tbl_user' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_user`.
in C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php (130)
in C:\xampp\htdocs\globaldebtmgm\index.php (13)
2011/12/11 22:39:43 [error] [exception.CDbException] exception 'CDbException' with message 'The table "tbl_user" for active record class "User" cannot be found in the database.' in C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php:2276
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(372): CActiveRecordMetaData->__construct(Object(User))
#1 C:\xampp\htdocs\YiiRoot\framework\web\CActiveDataProvider.php(72): CActiveRecord::model('User')
#2 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(130): CActiveDataProvider->__construct('User')
#3 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionIndex()
#4 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('')
#13 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user')
#14 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#15 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#16 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user
2011/12/11 22:41:06 [error] [exception.CException] exception 'CException' with message 'Property "User.updated_at" is not defined.' in C:\xampp\htdocs\YiiRoot\framework\base\CComponent.php:131
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(129): CComponent->__get('updated_at')
#1 C:\xampp\htdocs\YiiRoot\framework\web\helpers\CHtml.php(2063): CActiveRecord->__get('updated_at')
#2 C:\xampp\htdocs\YiiRoot\framework\web\helpers\CHtml.php(1791): CHtml::resolveValue(Object(User), 'updated_at')
#3 C:\xampp\htdocs\YiiRoot\framework\web\helpers\CHtml.php(1199): CHtml::activeInputField('text', Object(User), 'updated_at', Array)
#4 C:\xampp\htdocs\YiiRoot\framework\web\widgets\CActiveForm.php(568): CHtml::activeTextField(Object(User), 'updated_at', Array)
#5 C:\xampp\htdocs\globaldebtmgm\protected\views\user\_form.php(50): CActiveForm->textField(Object(User), 'updated_at')
#6 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\globaldebtmgm\protected\views\user\create.php(15): CController->renderPartial('_form', Array)
#10 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampp\htdocs...')
#11 C:\xampp\htdocs\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#13 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#14 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(78): CController->render('create', Array)
#15 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#16 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#17 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#23 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#25 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#26 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#27 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#28 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:42:09 [error] [exception.CException] exception 'CException' with message 'Property "User.modified_at" is not defined.' in C:\xampp\htdocs\YiiRoot\framework\base\CComponent.php:174
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(145): CComponent->__set('modified_at', Object(CDbExpression))
#1 C:\xampp\htdocs\YiiRoot\framework\validators\CDefaultValueValidator.php(42): CActiveRecord->__set('modified_at', Object(CDbExpression))
#2 C:\xampp\htdocs\YiiRoot\framework\validators\CValidator.php(192): CDefaultValueValidator->validateAttribute(Object(User), 'modified_at')
#3 C:\xampp\htdocs\YiiRoot\framework\base\CModel.php(152): CValidator->validate(Object(User), NULL)
#4 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(780): CModel->validate(NULL)
#5 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(72): CActiveRecord->save()
#6 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#7 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#8 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#10 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#12 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#13 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#14 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#16 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#17 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#18 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#19 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/create
2011/12/11 22:52:01 [error] [exception.CException] exception 'CException' with message 'Property "User.modified_at" is not defined.' in C:\xampp\htdocs\YiiRoot\framework\base\CComponent.php:174
Stack trace:
#0 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(145): CComponent->__set('modified_at', Object(CDbExpression))
#1 C:\xampp\htdocs\globaldebtmgm\protected\models\User.php(53): CActiveRecord->__set('modified_at', Object(CDbExpression))
#2 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(1059): User->beforeSave()
#3 C:\xampp\htdocs\YiiRoot\framework\db\ar\CActiveRecord.php(781): CActiveRecord->update(NULL)
#4 C:\xampp\htdocs\globaldebtmgm\protected\controllers\UserController.php(96): CActiveRecord->save()
#5 [internal function]: UserController->actionUpdate('3')
#6 C:\xampp\htdocs\YiiRoot\framework\web\actions\CAction.php(104): ReflectionMethod->invokeArgs(Object(UserController), Array)
#7 C:\xampp\htdocs\YiiRoot\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#8 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#11 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#13 C:\xampp\htdocs\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#15 C:\xampp\htdocs\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(277): CController->run('update')
#17 C:\xampp\htdocs\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/update')
#18 C:\xampp\htdocs\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#19 C:\xampp\htdocs\globaldebtmgm\index.php(13): CApplication->run()
#20 {main} REQUEST_URI=/globaldebtmgm/index.php?r=user/update&id=3
2011/12/11 22:34:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "clients/create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('clients/create')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=clients/create
2011/12/11 23:39:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "creditors/admin".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('creditors/admin')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=creditors/admin
2011/12/12 00:27:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "debtfinancialinfo".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtfinancialin...')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=debtfinancialinfo
2011/12/12 01:01:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "claim/admin".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('claim/admin')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=claim/admin
2011/12/12 01:05:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "additionalinfo/create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('additionalinfo/...')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=additionalinfo/create
2011/12/12 07:49:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "claim//create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('claim//create')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=claim//create
2011/12/12 07:49:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "claim/create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('claim/create')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=claim/create
2011/12/12 07:49:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "claim/create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('claim/create')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=claim/create
2011/12/12 07:49:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "claim//create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('claim//create')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=claim//create
2011/12/12 18:55:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "debtoradditonalinfo/create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtoradditonal...')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=debtoradditonalinfo/create
2011/12/12 23:31:05 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'globaldebtmgm.debtorsettlementoffer' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `DebtorSettlementOffer`.
in C:\xampplite\htdocs\project\globaldebtmgm\index.php (13)
2011/12/13 08:51:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "settlmentoffer/admin".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('settlmentoffer/...')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=settlmentoffer/admin
2011/12/13 08:51:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "settlementoffer/admin".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('settlementoffer...')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=settlementoffer/admin
2011/12/14 00:49:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "debtfinancialinfo/create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtfinancialin...')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=debtfinancialinfo/create
2011/12/14 20:03:00 [error] [exception.CException] exception 'CException' with message 'Property "CActiveForm.role" is not defined.' in C:\xampplite\htdocs\project\YiiRoot\framework\base\CComponent.php:131
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\helpers\CHtml.php(2063): CComponent->__get('role')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\web\helpers\CHtml.php(1397): CHtml::resolveValue(Object(CActiveForm), 'role')
#2 C:\xampplite\htdocs\project\YiiRoot\framework\web\widgets\CActiveForm.php(674): CHtml::activeDropDownList(Object(CActiveForm), 'role', Array, Array)
#3 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\user\_form.php(14): CActiveForm->dropDownList(Object(CActiveForm), 'role', Array)
#4 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#5 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#6 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#7 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\user\create.php(15): CController->renderPartial('_form', Array)
#8 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#9 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#10 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#11 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#12 C:\xampplite\htdocs\project\globaldebtmgm\protected\controllers\UserController.php(78): CController->render('create', Array)
#13 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CInlineAction.php(50): UserController->actionCreate()
#14 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#15 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#16 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#17 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#18 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#21 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#23 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('user/create')
#24 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#25 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#26 {main} REQUEST_URI=/index.php?r=user/create
2011/12/14 21:17:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "debtorinfo/create".' in C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php:281
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtorinfo/crea...')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#2 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#3 {main} REQUEST_URI=/index.php?r=debtorinfo/create
2011/12/14 23:06:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "subsub1" was not found.' in C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CViewAction.php:110
Stack trace:
#0 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CViewAction.php(121): CViewAction->resolveView('subsub1')
#1 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CAction.php(72): CViewAction->run()
#2 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(300): CAction->runWithParams(Array)
#3 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(278): CController->runAction(Object(CViewAction))
#4 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CViewAction), Array)
#5 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(277): CController->run('page')
#6 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('site/page')
#7 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#8 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#9 {main} REQUEST_URI=/index.php?r=site/page&view=subsub1
2011/12/15 01:11:18 [error] [exception.CException] exception 'CException' with message 'alwaysOpen is not a valid option' in C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php:265
Stack trace:
#0 C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php(149): EJqueryUiWidget::checkOptions(Array, Array)
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CComponent.php(153): EJqueryUiWidget->setOptions(Array)
#2 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWidgetFactory.php(162): CComponent->__set('options', Array)
#3 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(139): CWidgetFactory->createWidget(Object(DebtorController), 'application.ext...', Array)
#4 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(182): CBaseController->createWidget('application.ext...', Array)
#5 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\_form.php(36): CBaseController->beginWidget('application.ext...', Array)
#6 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#7 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#8 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#9 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\create.php(15): CController->renderPartial('_form', Array)
#10 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#11 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#12 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#13 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#14 C:\xampplite\htdocs\project\globaldebtmgm\protected\controllers\DebtorController.php(78): CController->render('create', Array)
#15 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CInlineAction.php(50): DebtorController->actionCreate()
#16 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#17 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#20 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#23 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#25 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtor/create')
#26 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#27 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#28 {main} REQUEST_URI=/index.php?r=debtor/create
2011/12/15 01:12:19 [error] [exception.CException] exception 'CException' with message 'alwaysOpen is not a valid option' in C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php:265
Stack trace:
#0 C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php(149): EJqueryUiWidget::checkOptions(Array, Array)
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CComponent.php(153): EJqueryUiWidget->setOptions(Array)
#2 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWidgetFactory.php(162): CComponent->__set('options', Array)
#3 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(139): CWidgetFactory->createWidget(Object(DebtorController), 'application.ext...', Array)
#4 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(182): CBaseController->createWidget('application.ext...', Array)
#5 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\_form.php(36): CBaseController->beginWidget('application.ext...', Array)
#6 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#7 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#8 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#9 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\create.php(15): CController->renderPartial('_form', Array)
#10 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#11 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#12 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#13 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#14 C:\xampplite\htdocs\project\globaldebtmgm\protected\controllers\DebtorController.php(78): CController->render('create', Array)
#15 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CInlineAction.php(50): DebtorController->actionCreate()
#16 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#17 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#20 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#23 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#25 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtor/create')
#26 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#27 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#28 {main} REQUEST_URI=/index.php?r=debtor/create
2011/12/15 01:12:23 [error] [exception.CException] exception 'CException' with message 'alwaysOpen is not a valid option' in C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php:265
Stack trace:
#0 C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php(149): EJqueryUiWidget::checkOptions(Array, Array)
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CComponent.php(153): EJqueryUiWidget->setOptions(Array)
#2 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWidgetFactory.php(162): CComponent->__set('options', Array)
#3 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(139): CWidgetFactory->createWidget(Object(DebtorController), 'application.ext...', Array)
#4 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(182): CBaseController->createWidget('application.ext...', Array)
#5 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\_form.php(36): CBaseController->beginWidget('application.ext...', Array)
#6 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#7 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#8 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#9 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\create.php(15): CController->renderPartial('_form', Array)
#10 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#11 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#12 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#13 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#14 C:\xampplite\htdocs\project\globaldebtmgm\protected\controllers\DebtorController.php(78): CController->render('create', Array)
#15 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CInlineAction.php(50): DebtorController->actionCreate()
#16 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#17 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#20 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#23 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#25 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtor/create')
#26 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#27 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#28 {main} REQUEST_URI=/index.php?r=debtor/create
2011/12/15 01:12:26 [error] [exception.CException] exception 'CException' with message 'alwaysOpen is not a valid option' in C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php:265
Stack trace:
#0 C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php(149): EJqueryUiWidget::checkOptions(Array, Array)
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CComponent.php(153): EJqueryUiWidget->setOptions(Array)
#2 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWidgetFactory.php(162): CComponent->__set('options', Array)
#3 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(139): CWidgetFactory->createWidget(Object(DebtorController), 'application.ext...', Array)
#4 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(182): CBaseController->createWidget('application.ext...', Array)
#5 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\_form.php(36): CBaseController->beginWidget('application.ext...', Array)
#6 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#7 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#8 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#9 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\create.php(15): CController->renderPartial('_form', Array)
#10 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#11 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#12 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#13 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#14 C:\xampplite\htdocs\project\globaldebtmgm\protected\controllers\DebtorController.php(78): CController->render('create', Array)
#15 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CInlineAction.php(50): DebtorController->actionCreate()
#16 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#17 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#20 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#23 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#25 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtor/create')
#26 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#27 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#28 {main} REQUEST_URI=/index.php?r=debtor/create
2011/12/15 01:13:01 [error] [exception.CException] exception 'CException' with message 'alwaysOpen is not a valid option' in C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php:265
Stack trace:
#0 C:\xampplite\htdocs\project\globaldebtmgm\protected\extensions\jui\EJqueryUiWidget.php(149): EJqueryUiWidget::checkOptions(Array, Array)
#1 C:\xampplite\htdocs\project\YiiRoot\framework\base\CComponent.php(153): EJqueryUiWidget->setOptions(Array)
#2 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWidgetFactory.php(162): CComponent->__set('options', Array)
#3 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(139): CWidgetFactory->createWidget(Object(DebtorController), 'application.ext...', Array)
#4 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(182): CBaseController->createWidget('application.ext...', Array)
#5 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\_form.php(24): CBaseController->beginWidget('application.ext...', Array)
#6 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#7 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#8 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#9 C:\xampplite\htdocs\project\globaldebtmgm\protected\views\debtor\create.php(15): CController->renderPartial('_form', Array)
#10 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(119): require('C:\xampplite\ht...')
#11 C:\xampplite\htdocs\project\YiiRoot\framework\web\CBaseController.php(88): CBaseController->renderInternal('C:\xampplite\ht...', Array, true)
#12 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(866): CBaseController->renderFile('C:\xampplite\ht...', Array, true)
#13 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(779): CController->renderPartial('create', Array, true)
#14 C:\xampplite\htdocs\project\globaldebtmgm\protected\controllers\DebtorController.php(78): CController->render('create', Array)
#15 C:\xampplite\htdocs\project\YiiRoot\framework\web\actions\CInlineAction.php(50): DebtorController->actionCreate()
#16 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(300): CInlineAction->runWithParams(Array)
#17 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#18 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilter.php(41): CFilterChain->run()
#19 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(1144): CFilter->filter(Object(CFilterChain))
#20 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CInlineFilter.php(59): CController->filterAccessControl(Object(CFilterChain))
#21 C:\xampplite\htdocs\project\YiiRoot\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#22 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(283): CFilterChain->run()
#23 C:\xampplite\htdocs\project\YiiRoot\framework\web\CController.php(257): CController->runActionWithFilters(Object(CInlineAction), Array)
#24 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(277): CController->run('create')
#25 C:\xampplite\htdocs\project\YiiRoot\framework\web\CWebApplication.php(136): CWebApplication->runController('debtor/create')
#26 C:\xampplite\htdocs\project\YiiRoot\framework\base\CApplication.php(158): CWebApplication->processRequest()
#27 C:\xampplite\htdocs\project\globaldebtmgm\index.php(13): CApplication->run()
#28 {main} REQUEST_URI=/index.php?r=debtor/create
